src/codegen/luacg.cpp

```diff
@@ -866,7 +875,7 @@ wxString LuaCodeGenerator::GetCode(PObjectBase obj, wxString name, bool silent /
 
 	if (!m_strUITable.empty())
 	{
-		_template.Replace(wxT("#utbl"), m_strUITable + wxT("."));
+		_template.Replace(wxT("#utbl"), wxT("self."));
```

Comment:
And if the table name is empty, the empty string will be used, doesn't look right